lte/protos/oai/nas_state.proto

```diff
@@ -475,4 +479,68 @@ message EmmContext {
   NewAttachInfo new_attach_info = 67;
   UeAdditionalSecurityCapability ue_additional_security_capability = 68;
   // TODO: add remaining emm_context elements
+
+  // 5G Guti
+  Guti_m5 m5_guti = 91;
+  Guti_m5 m5_old_guti = 92;
+  // 5G proto
+  map<uint32,SmfContext> smf_context_m =93;
+
+  }
+
+  // guti_m5_t
+  message Guti_m5 {
+    bytes plmn = 1;
+    uint32 amf_regionid = 2;
+    uint32 amf_set_id = 3;
+    uint32 amf_pointer = 4;
+    uint32 m_tmsi = 5;
+  }
+
+  message Smf_Proc_Data {
+    uint32 pdu_session_id = 1;
+    uint32 pti = 2;
+    uint32 message_type = 3;
+    uint32 max_uplink = 4;
+    uint32 max_downlink = 5;
+    uint32 pdu_session_type = 6;
```

Comment:
Enum as used in session_manager.proto

Reply:
For this change, the application code change requested in #10853 is needed. The `enum PduSessionType` should be redefined in nas_state.proto similar to how it is defined in session_manager.proto, right?

Reply:
The changes here can happen in parallel with the changes in #10853 , and the PR with the state converter logic can  stitch them together.

Reply:
Okay sure, made the change.